signup.js

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { SafeAreaView, View, StyleSheet,Image } from 'react-native'
+import {DefaultTheme,configureFonts,Provider,Button, TextInput,Text} from 'react-native-paper'
+
+const fontConfig = {
+    default: {
+      regular: {
+        //Decide on main font
+      },
+      medium: {
+        //Decdie on medium font
+      },
+      light: {
+        //Decide on light font
+      },
+  
+    },
+  };
+  
+  const theme = {
```

Comment:
I haven't looked at theme yet, could you just briefly tell me what you're using it for here. Also @ctperry0301 is this something we only want once in our app since we have it currently in App.js

Reply:
Yeah. Everything related to Theme should only be in the App.js file. The theme will be applied everywhere across the site that is nested within <Provider theme={theme}> in the App.js file.

Reply:
Only had that there temporarily so that I could see my code, I set it as the appentry and needed it to be inside the theme. Taking it out now it shouldn't have been included in the PR